src/splitAvgMinDistance.test.js

```diff
@@ -0,0 +1,22 @@
+const {splitAvgMinDistance} = require("./splitAvgMinDistance.js");
+
+test("Return values of splitAvgMinDistance", () => {
+    let cTwoSix = [
+        [9 / 11, 7 / 11, 9 / 11, 3 / 11],
```

Comment:
Lav i stedet en funktion der dividere alle elementer i et array med 11 og returner det array

Reply:
vi har bestemt os for at lade være siden det bare er den enkelte test